ROADMAP.md

```diff
@@ -0,0 +1,25 @@
+# Community-Driven Roadmap
+
+Shipwright's detailed roadmap can be found on the
+[Shipwright Overview GitHub Project](https://github.com/orgs/shipwright-io/projects/6). All
+repositories are currently tied to the release cycle of the core
+[build](https://github.com/shipwright-io/build) sub-project. Shipwright aims to issue minor
+releases quarterly, using the same major and minor semantic version across repositories.
+
+## 2025 Release Schedule
+
+Below is the tentative release schedule for 2025. Actual release dates may vary based on community
+availability and release stability.
```

Comment:
From community meeting - need to clarify meaning of Kubernetes and Tekton Pipelines version below.